scripts/dependency-graph-generator/index.js

```diff
@@ -0,0 +1,174 @@
+const graphviz = require('graphviz');
+const parser = require('dotparser');
+const readFileSync = require('fs').readFileSync;
+const spawnSync = require('child_process').spawnSync;
+const findGitRoot = require('../monorepo/index').findGitRoot;
+const getDevDependencies = require('./getDevDependencies');
+const path = require('path');
+const os = require('os');
+
+const dotFilePath = path.resolve(__dirname, 'repo-graph.dot');
+let ignoreDevDependencies = [];
+
+/**
+ * Pick a package in the repository and generate its dependency graph
+ *
+ * For this utility to work you will actually need to install graphviz on your machine
```

Comment:
Not a big deal (especially since you've already written this) but I wonder if there's something we could use that doesn't require the separate graphviz install? I know there are package dependency graph generator packages that work without it.

Reply:
Perhaps, but I am stupid and I found the first one that was lerna based :)